source/macos/AdaptiveCards/AdaptiveCards/Base/ACRImageProperties.swift

```diff
@@ -37,7 +37,7 @@ class ACRImageProperties: NSObject {
                                                       height: pixelHeight,
                                                       with: config,
                                                       explicitDimensions: hasExplicitDimensions)
-        updateContentSize(size: image.size)
+        updateContentSize(size: image?.size ?? CGSize.zero)
```

Comment:
Just curious. Do we really need to assume .zero for nil case, or we  can just ignore updating the contentSize.

Reply:
Updating the content size is depends on multiple factors such as explicit dimensions, ACS image sizes (small, medium and large) etc 
So when the image size is unknown or nil during the initial init of ImageRenderer, we can treat it as CGSize.zero. 
`updateContentSize()` has other checks in place for this scenario